src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -84,57 +85,66 @@ object Read extends Logging {
    */
   def apply(record: SAMRecord): Read = {
 
-    val isMapped = (
+    val isMapped =
       !record.getReadUnmappedFlag &&
       record.getReferenceName != null &&
       record.getReferenceIndex >= SAMRecord.NO_ALIGNMENT_REFERENCE_INDEX &&
       record.getAlignmentStart >= 0 &&
-      record.getUnclippedStart >= 0)
+      record.getUnclippedStart >= 0
 
-    val sampleName = (if (record.getReadGroup != null && record.getReadGroup.getSample != null) {
-      record.getReadGroup.getSample
-    } else {
-      "default"
-    }).intern
-
-    val read = if (isMapped) {
-      val result = MappedRead(
-        record.getReadName,
-        record.getReadString.getBytes,
-        record.getBaseQualities,
-        record.getDuplicateReadFlag,
-        sampleName.intern,
-        record.getReferenceName.intern,
-        record.getMappingQuality,
-        record.getAlignmentStart - 1,
-        cigar = record.getCigar,
-        failedVendorQualityChecks = record.getReadFailsVendorQualityCheckFlag,
-        isPositiveStrand = !record.getReadNegativeStrandFlag,
-        isPaired = record.getReadPairedFlag
-      )
+    val sampleName =
+      (if (record.getReadGroup != null && record.getReadGroup.getSample != null)
```

Comment:
I like omitting curly-braces for expressions like this; we shouldn't accumulate any / a lot of state in inline blocks, so the default should be single expressions, and omitting curly-braces reduces clutter significantly imho.